src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -253,6 +253,22 @@ public static string CleanTitle(string title)
             return title;
         }
 
+        public static string TitleThe(string title)
+        {
+            string[] prefixes = { "The ", "An ", "A " };
+            foreach (string prefix in prefixes)
+            {
+                int prefix_length = prefix.Length;
+                if (prefix.ToLower() == title.Substring(0, prefix_length).ToLower())
```

Comment:
And we got something called `string.StartsWith(..., StringComparison.OrdinalIgnoreCase)`